probe_scraper/parsers/repositories.py

```diff
@@ -22,10 +22,17 @@ def __init__(self, name, definition):
         self.notification_emails = definition.get("notification_emails")
         self.app_id = definition.get("app_id")
         self.metrics_file_paths = definition.get("metrics_files", [])
+        self.library_names = definition.get("library_names", None)
+        self.dependencies_url = definition.get("dependencies_url", None)
+        self.dependencies_format = definition.get("dependencies_format", None)
+        self.dependencies_files = definition.get("dependencies_files", [])
 
     def get_metrics_file_paths(self):
         return self.metrics_file_paths
 
+    def get_change_files(self):
```

Comment:
There is some unnecessary work now happening, where metrics and dependencies are being read in commits where they didn't change. This isn't a big deal but could be a future point of optimization.

Reply:
That's a good point.  I'm not seeing a *simple* way to address that now, since it tracks dependencies and probes together.  Are you comfortable with coming back to this later if it becomes a performance problem?  (I don't think any of these files change terribly often, relative to other files in the repo...)